api/src/main/scala/hmda/api/http/HttpApi.scala

```diff
@@ -25,6 +41,37 @@ trait HttpApi extends HmdaApiProtocol {
           ToResponseMarshallable(status)
         }
       }
-    }
+    } ~
+      path("upload") {
+        post {
+          val processingActor = system.actorOf(ProcessingActor.props)
+          entity(as[Multipart.FormData]) { formData =>
+            val uploaded: Future[Done] = formData.parts.mapAsync(1) {
```

Comment:
Until we measure things under load we don't know what the optimum number is, most examples start with a value of 1 and take it from there. Not sure if this will influence things that much since the process is I/O bound